src/app/auth/forgot-password/forgot-password.component.html

```diff
@@ -0,0 +1,20 @@
+<div
+  class="card-container"
+  fxLayout="row wrap"
+  fxLayoutAlign="center center"
+  style="height: 70%;"
```

Comment:
It's always good to practise to have CSS rules in a .scss file. Avoid in-line CSS.

Reply:
@Bbaale15 I have implemented the feedback that you left on the PR